src/datamodel/character/test.rs

```diff
@@ -137,6 +137,28 @@ fn test_minion_of_mordred_cannot_see_loyal_servant_of_arthur() {
 }
 
 //TEST MORGANA
```

Comment:
Wouldn't it be better to have a structure in the test files similar to the source code? Instead of having these comments, we could have one file per character, as we do in the source folder.